internal/directives/schemas/yaml-parse-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "YAMLParseConfig",
+
+  "definitions": {
+    "yamlParse": {
```

Comment:
As this appears to be equal to `jsonParse` in the other schema, I wonder if we would benefit from a common type that's shared between the two schemas. By doing this, I suspect we can share much more code between the two parsers.

Reply:
Deferred as it is non-blocking.